skrf/circuit.py

```diff
@@ -143,7 +143,7 @@ def _get_nx():
     def __init__(self,
                  connections: list[list[tuple[Network, int]]],
                  name: str | None = None,
-                 auto_reduce: bool = False) -> None:
+                 auto_reduce: bool = False, **kwargs) -> None:
```

Comment:
```suggestion
                 *,
                 auto_reduce: bool = False, **kwargs) -> None:
```

I think we should not allow `auto_reduce` as positional argument.

Reply:
Modified as suggested